build_container.sh

```diff
@@ -11,10 +11,16 @@ DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y \
     libssl-dev tzdata cmake g++ pkg-config jq libcurl4-openssl-dev libelf-dev \
     libdw-dev binutils-dev libiberty-dev make \
     cpio bc flex bison wget xz-utils fakeroot \
+    cmake cmake-data \
+    build-essential libjsoncpp25 librhash0 make \
     autoconf autoconf-archive automake libtool \
     libclang-dev iproute2 \
     libasound2 libasound2-dev \
     libepoxy0 libepoxy-dev \
+    libdrm2 libdrm-dev \
+    libglm-dev libstb-dev \
+    libvirglrenderer-dev libvirglrenderer1 \
+
```

Comment:
We should remove this line blank to fix the issue in the CI

Reply:
ahh yes! thank you